ddtrace/constants.py

```diff
@@ -1 +1,2 @@
 FILTERS_KEY = 'FILTERS'
+SAMPLING_PRIORITY_KEY = 'sampling.priority'
```

Comment:
While our initial implementation is still "experimental" and is likely to change a lot in the future, what do we think about using a non-finale key for that?

Something like `_sampling_priority_v1 `, that way upgrades of our sampling logic will be much simpler.

The usage of the exact same OT key isn't important right, and it doesn't especially make sense anyway since we aren't compatible / the exact meaning of this value isn't properly defined by OT.